php.ini-development

```diff
@@ -1957,6 +1957,16 @@ ldap.max_links = -1
 ; Useful for internal debugging only.
 ;opcache.protect_memory=0
 
+; Allows calling OPcache API functions only from PHP scripts which path is
+; started from specified string. The default "" means no restriction
+;opcache.restrict_api=
+
+; Mapping base of shared memory segments (for Windows only). All the PHP
+; processes have to map shared memory into the same address space. This
+; directive allows to manually fix the "Unable to reattach to base address"
+; errors.
+;opcache.mmap_base=
```

Comment:
; no restriction
opcache.restrict_api=

; try to select some available address
opcache.mmap_base=


On Sun, Feb 1, 2015 at 5:47 PM, Andrea Faulds <notifications@github.com>
wrote:

> In php.ini-development
> <https://github.com/php/php-src/pull/1040#discussion_r23898013>:
>
> > @@ -1957,6 +1957,16 @@ ldap.max_links = -1
> >  ; Useful for internal debugging only.
> >  ;opcache.protect_memory=0
> >
> > +; Allows calling OPcache API functions only from PHP scripts which path is
> > +; started from specified string. The default "" means no restriction
> > +;opcache.restrict_api=
> > +
> > +; Mapping base of shared memory segments (for Windows only). All the PHP
> > +; processes have to map shared memory into the same address space. This
> > +; directive allows to manually fix the "Unable to reattach to base address"
> > +; errors.
> > +;opcache.mmap_base=
>
> What does this blank default mean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/php/php-src/pull/1040/files#r23898013>.
>